### ISSUE TITLE
CSRF with Fluid

### ISSUE THREAD

**Issue by ronfcodes** [sebastienros/fluid#350]
Hey, I have an asp.net core application and I'm using fluid v2.  I'm trying to find out is there a mechanism to deal with csrf?  Using a cshtml page I'm able to use the @Html.AntiForgeryToken() with a form.  When I checked the page elements with a liquid file there isn't a token so I'm wondering if there are any ways to handle that?

**Comment by sebastienros** [comment]
I forgot to mention, we do that in Orchard Core, so this could be copy-pasted:

https://github.com/OrchardCMS/OrchardCore/blob/0a86edff420cd693578ec2e29e6576bd90888fe4/src/OrchardCore/OrchardCore.DisplayManagement.Liquid/LiquidViewParser.cs#L22

https://github.com/OrchardCMS/OrchardCore/blob/0a86edff420cd693578ec2e29e6576bd90888fe4/src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Tags/AntiForgeryTokenTag.cs#L13

**Comment by ronfcodes** [comment]
Thanks! Does the attribute still need to be applied to the controller using this method?

**Comment by sebastienros** [comment]
Yes, the attribute is to validate on the server side the value generated by this tag on the client side (and in the cookies)

**Comment by ronfcodes** [comment]
Awesome, I will take this and run with it thanks!
